.github/workflows/push-broker.yaml

```diff
@@ -0,0 +1,37 @@
+---
+jobs:
+  build-push:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: 'actions/checkout@v4'
+    - id: 'metadata'
+      uses: 'docker/metadata-action@v5'
+      with:
+        images: 'econialabs/emojicoin-broker'
+        tags: |
+          type=ref,event=branch
+          type=semver,pattern={{version}}
```

Comment:
As I recall this won't work becayse `type=semver` requires a tag of semver form, not `emojicoin-indexer-broker-<semver>`

- [ ] Update to replicate `cloud-infra` example